lib/terrapin/command_line.rb

```diff
@@ -50,16 +50,14 @@ def best_runner
 
     def initialize(binary, params = "", options = {})
       @binary = binary.dup
-      if params.nil?
-        raise ArgumentError, "2nd argument to CommandLine.new should be a " \
-          "string representing the command line options"
-      end
-      @params = params.dup
+      @params = params.to_s.dup
+
       if options.nil?
```

Comment:
Ok, but if we're going to do this, I'd like you go move this to the very beginning so then we can keep the first three assignment lines together and with the same formatting as the rest.